lib/dpl/ctx/bash.rb

```diff
@@ -490,6 +490,7 @@ def mv(src, dest)
       # Writes the given content to the given file path
       def write_file(path, content, chmod = nil)
         path = File.expand_path(path)
+        FileUtils.mkdir_p(File.dirname(path))
```

Comment:
This probably needs to go to a separate PR. It doesn't look related.

Reply:
right, not sure why this ended up here